### PR TITLE
Nebula12b

### PR DESCRIPTION
**Pull request by yiancar**
## Description

Successor of Nebula12.
Similar story as Nebula68b:)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/spaceholdings/nebula12b/config.h [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTICROSS
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTINEXUS
+#define ENABLE_RGB_MATRIX_SPLASH
+#define ENABLE_RGB_MATRIX_MULTISPLASH
+#define ENABLE_RGB_MATRIX_SOLID_SPLASH
+#define ENABLE_RGB_MATRIX_SOLID_MULTISPLASH
+
+// // VIA lighting is handled by the keyboard-level code
+#define VIA_CUSTOM_LIGHTING_ENABLE
```
Comment (drashna) [inline keyboards/spaceholdings/nebula12b/config.h:134]: This isn't needed anymore.
```suggestion
```

File: keyboards/spaceholdings/nebula12b/nebula12b.c [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            eeconfig_update_rgb_matrix();
+            break;
+        }
+        default: {
+            *command_id         = id_unhandled;
+            break;
+        }
+    }
+}
+#endif
```
Comment (drashna) [inline keyboards/spaceholdings/nebula12b/nebula12b.c:111]:
```suggestion
```

**Comment by yiancar** [comment]
Ah yes nice!
One addition for the via code tho (in via.c) would be to scale brightness with MAX_BRIGHTNESS variable ;) I can do the PR if you like.

File: keyboards/spaceholdings/nebula12b/readme.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* Hardware Supported: A micropad with ATMEGA32U4
+* Hardware Availability: https://spaceholdings.net/
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make spaceholdings/nebula12b:via
```
Comment (fauxpark) [inline keyboards/spaceholdings/nebula12b/readme.md:15]:
```suggestion
    make spaceholdings/nebula12b:default
```

File: keyboards/spaceholdings/nebula12b/nebula12b.c [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "nebula12b.h"
+#include <lib/lib8tion/lib8tion.h>
```
Comment (fauxpark) [inline keyboards/spaceholdings/nebula12b/nebula12b.c:18]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
> Ah yes nice! One addition for the via code tho (in via.c) would be to scale brightness with MAX_BRIGHTNESS variable ;) I can do the PR if you like.

Hmm, I'm not sure. In theory, even if you set the val to a higher value, it won't use past that. But I'm not sure that via will re-read and report that correctly?  May need some testing/etc.

**Comment by yiancar** [comment]
> > Ah yes nice! One addition for the via code tho (in via.c) would be to scale brightness with MAX_BRIGHTNESS variable ;) I can do the PR if you like.
> 
> Hmm, I'm not sure. In theory, even if you set the val to a higher value, it won't use past that. But I'm not sure that via will re-read and report that correctly? May need some testing/etc.

Yes that is correct, essentially by doing a scale operation (as I had done here) the brightness dial will not jump around in via. Currently lets say max brightness is at 150, half the dial does nothing. You slide it to 100% and it will jump back to 50% after the refresh.

**Comment by yiancar** [comment]
Can we get this merged please?:)

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**